mem-13-Q.xml

```diff
@@ -5861,16 +5963,14 @@ The slang meaning of {chong:v:2,slang}, when applied to a person, is equivalent
       <column name="definition_de">ein Spiel spielen</column>
       <column name="definition_fa">بازی کردن [AUTOTRANSLATED]</column>
       <column name="definition_sv">spela</column>
-      <column name="definition_ru"></column>
+      <column name="definition_ru">сыграть в игру</column>
       <column name="definition_zh_HK">玩個遊戲 [AUTOTRANSLATED]</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{reH:v}, {ghet:v}, {Quj:n}</column>
       <column name="notes">The difference between this and {reH:v} is that this refers to structured play, whereas {reH:v:nolink} refers to unstructured play.[2]</column>
       <column name="notes_de"></column>
-      <column name="notes_fa"></column>
```

Comment:
These shouldn't have been moved.

Reply:
fixed

Reply:
Just so you know, no changes made after the PR has already been merged will be pulled into master. So there's no need for you to fix these (as I've already fixed them when I did the merge).